docs/source/features/azureml_integration.md

```diff
@@ -0,0 +1,104 @@
+# Azure ML integration
+
+This documents outlines the integrations between Olive and Azure Machine Learning. Discover how to optimize your Azure Machine Learning assets within Olive.
+
+## Olive Core
+### Using AzureML registered model
+You can run Olive workflow with your AML workspace registered model. In the input model section, define the model config as:
+```
+"model_path": {
+    "type": "azureml_model",
+    "config": {
+        "name": "<model_name>",
+        "version": "<model_version>"
+    }
+ }
+```
+Olive will automatically download the model and run the workflow with this model as input model.
+
+### Using model stored in AzureML datastore
+You can specify your model path from an AzureML datastore as:
+```
```

Comment:
Do you mean models from the curated catalogue?

Reply:
No, this is from user datastore. They can store their model in their datastore.